### PR TITLE
Add templates for almalinux 8 and debian 11

### PR DESCRIPTION
**Pull request by gduperrey**
Adding two templates for AlmaLinux 8 and Debian 11.

### CODE REVIEW

File: SPECS/guest-templates-json.spec [hunk 1]
```diff
@@ -183,6 +187,8 @@ fi
 %{templatedir}/other-install-media.json
 
 %changelog
+* Mon Nov 22 2021 Gael Duperrey <gduperrey@vates.fr> - 1.9.4-1.2
+- Add template for AlmaLinux 8 and Debian 11 Bulleyes
```
Comment (stormi) [inline SPECS/guest-templates-json.spec]:
```suggestion
- Add template for AlmaLinux 8 and Debian 11 Bullseye

```

File: SPECS/guest-templates-json.spec [hunk 2]
```diff
@@ -1,11 +1,14 @@
 Name:    guest-templates-json
 Summary: Creates the default guest templates
 Version: 1.9.4
-Release: 1.1%{?dist}
+Release: 1.2%{?dist}
 License: BSD
 
 Source0: https://code.citrite.net/rest/archive/latest/projects/XS/repos/guest-templates-json/archive?at=v1.9.4&format=tar.gz&prefix=guest-templates-json-1.9.4#/guest-templates-json-1.9.4.tar.gz
 
+#XCP-ng patches
```
Comment (stormi) [inline SPECS/guest-templates-json.spec]:
```suggestion
# XCP-ng patches
```

**Review by stormi — CHANGES_REQUESTED** [review]

**Comment by stormi** [comment]
The target branch for the PR should be 8.2 as this is meant to be an update to the stable release of XCP-ng.

**Comment by gduperrey** [comment]
[build koji ok](https://koji.xcp-ng.org/taskinfo?taskID=20566)

**Pull request merged**